lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -40,6 +41,7 @@ import { Constants,
     PersistentCacheKeys,
     ErrorCacheKeys,
 } from "./utils/Constants";
+import uuid from "uuid";
```

Comment:
Any reason we don't use the in house guid creator?

Reply:
great call, fixing

Reply:
updated